### ISSUE TITLE
Random order of variables in init.sh

### ISSUE THREAD

**Issue by ktf** [alisw/alibuild#210]
some `init.sh` scripts generated by AliBuild/AliDist have a random order for variable initialization.

I noticed for instance the following wrong order in GEANT4/.../init.sh :

```
export G4LEDATA="$G4INSTALL_DATA/data/G4EMLOW6.41"
export G4INSTALL_DATA="$GEANT4_ROOT/share/..."
```

**Comment by ktf** [comment]
Replaces alisw/alidist#288

**Comment by dberzano** [comment]
Fixed in #466.
